lib/henkei/configuration.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# Henkei monkey patch for configuration support
+class Henkei
+  class << self
+    attr_accessor :configuration
+
+    def configure
```

Comment:
I would suggest a slightly different pattern here. Something like:
```
  def self.configuration
    @configuration ||= Configuration.new
  end

  def self.configure
    yield(configuration)
  end
```
Benefits are:
1. you don't need to bootstrap the configuration (ie the call to `Henkei.configure`)
2. in reality I suspect you really don't want the configuration (the class object) to be overwritten. Could use `attr_reader` but splitting it out solves both issues.